src/pudl/transform/ferc1.py

```diff
@@ -124,6 +124,117 @@ class Config:
 ################################################################################
 
 
+################################################################################
+# FERC 1 transform helper functions. Probably to be integrated into a class
+# below as methods or moved to a different module once it's clear where they belong.
```

Comment:
Any thoughts on where these "helper functions" should really end up living?

Reply:
i can imagine keeping them here but structuring them more like the transformers in the classes module? but idk that is my first reflex. let me read through them.

Reply:
oh these are the glue mapping helper functions? like they are used to update the row literal (dbf) to column (xbrl) map csv? I would maybe even put these in the `glue` directory instead of here.

Reply:
My first impulse was also to put them in with the other inter-dataset `glue` mapping helper functions (in their own ferc to ferc module) since it's very analogous to the kind of stuff we do with the FERC to EIA to PUDL mapping, but it's within a dataset (FERC 1 XBRL vs. FERC 1 DBF). 

My intuition is that the one function which retrieves the already mapped data should probably be used to read a CSV that's used to instantiate a transform param. So it would show up in the `params.ferc1` module

Reply:
yea that sounds right. There are many FERC1 only (dbf-xbrl) glue functions in the `glue.ferc_eia` module that could be brought over into a `glue.ferc` module

Reply:
Okay, can we bike-rack that for next week?

Reply:
@cmgosnell says yes. Next week.